internal/pkg/prune/prune.go

```diff
@@ -143,3 +143,28 @@ func (o *Prune) deleteObject(ctx context.Context, gvk schema.GroupVersionKind,
 	}
 	return obj, nil
 }
+
+// GetPruneResources finds the resource used for pruning from a slice of resources
+// by looking for a special annotation in the resource
+// inventory.InventoryAnnotation
+func GetPruneResources(resources []*unstructured.Unstructured) (*unstructured.Unstructured, error) {
```

Comment:
This function should have a unit test.

Reply:
I moved this function to `resourceconfig` package and added a unit test.